Backend/app/rider-platform/rider-app/Main/src/Storage/Tabular/Merchant/MerchantServiceUsageConfig.hs

```diff
@@ -52,6 +52,7 @@ mkPersist
       whatsappProvidersPriorityList (PostgresList WhatsappService)
       updatedAt UTCTime
       createdAt UTCTime
+      getDistancesOSRM MapsService
```

Comment:
Can you move your field before `createdAt` and `updatedAt`. This two fields should be the last in all types